examples/linq/1.the_basics.ipynb

```diff
@@ -844,7 +844,7 @@
     "# Option2: Assume quantum circuit simulation was performed separately (by this package or different services)\n",
     "freqs, _ = sim.simulate(c)  # circuit c must be consistent with operator, regarding measurement along non-z axes\n",
     "term, coef = tuple(op.terms.items())[0]  # This yields ((0, 'Z'),), 1.0\n",
-    "expval2 = coef * Simulator.get_expectation_value_from_frequencies_oneterm(term, freqs)\n",
+    "expval2 = coef * sim.get_expectation_value_from_frequencies_oneterm(term, freqs)\n",
```

Comment:
We've known this to be a horrendously long name for a while. I don't want to bother with it in this PR because I think we'd do it with a proper deprecation after Alexandre's PR has been merged. But I think `expval` or something is better than `get_expectation_value` and we should have something shorter than the one above.